fe/fe-core/src/main/java/org/apache/doris/nereids/stats/JoinEstimation.java

```diff
@@ -41,10 +43,11 @@ public class JoinEstimation {
      * Do estimate.
      */
     public static StatsDeriveResult estimate(StatsDeriveResult leftStats, StatsDeriveResult rightStats,
```

Comment:
Maybe passing a whole join node as the parameter is more clear, rather than properties in join.